src/components/Sponsors/ThankYouSection.js

```diff
@@ -0,0 +1,193 @@
+            marginTop: 'calc(40vw + 200px) ',
+            left: 'calc(20vw + 50px)',
+        },
+    },
+}));
+
+export default function ThankYou() {
+
+    const classes = useStyles();
+
+    return (
+
+        <Grid className={classes.container} alignItems="center" justify="center">
+
+            <Grid item xs={12} sm={6} lg={8} xl={12} className={classes.starsStyle}>
+                <SVGLoad image={Stars} alt="sponsorStars" />
+            </Grid>
+
+            <Grid item xs={5} sm={4} lg={8} xl={12} className={classes.handsStyle}>
+                <SVGLoad image={Hands} alt="sponsorStars" />
+            </Grid>
+
+            <Grid item xs={12} sm={12} lg={8} xl={10} className={classes.blobStyle}>
+
+                <svg viewBox="0 0 1056 344" fill="none" xmlns="http://www.w3.org/2000/svg">
+
+                    <path fill-rule="evenodd" clip-rule="evenodd" d="M692.553 273.274C834.991 166.805 1067.12 185.273 1055.09 95.7201C1044.7 18.4531 382.414 -27.4718 259.28 20.5836C144.282 65.4637 82.5093 82.6956 20.5314 184.331C-35.1913 275.709 37.0658 300.949 99.3671 322.635C175.328 349.075 566.737 367.318 692.553 273.274Z" fill="#E8D1FF" fill-opacity="0.4"/>
+        
+                    <foreignObject x='434' y='17' width="140" height="44" className={classes.headerText}> 
+                        Thank you!
+                    </foreignObject>
+
+                    <foreignObject x='168' y='76' className={classes.bodyText} width="672" height="70">
+                        <p className={classes.alignBodyText}>We are exceedingly grateful to our sponsors, who allow us to continue to </p>
+                        <p className={classes.alignBodyText}>offer life-changing tech mentoring through our program. </p>
+                    </foreignObject>
+
+                    <foreignObject x='168' y='155' className={classes.bodyText} width="719" height="120">
+                        <p className={classes.alignBodyText}>Our work would not have been possible without the overwhelming positive </p>
+                        <p className={classes.alignBodyText}>and creative community of Civictech Toronto. Thank you to the amazing </p>
+                        <p className={classes.alignBodyText}>volunteers, sponsors, and community partners who help make this space a </p>
+                        <p className={classes.alignBodyText}>reality so projects like ours can begin. </p>
+                    </foreignObject>
+
+                </svg>
+
+                <svg className={classes.mobileBlobStyle} height="360" viewBox="0 0 360 185" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path fill-rule="evenodd" clip-rule="evenodd" d="M281.222 146.894C358.047 89.4647 483.251 99.4261 476.757 51.1214C471.154 9.44378 113.946 -15.328 47.5323 10.593C-14.4929 34.8012 -47.8104 44.096 -81.2387 98.9181C-111.293 148.207 -72.3207 161.822 -38.7179 173.519C2.25214 187.78 213.362 197.621 281.222 146.894Z" fill="#E8D1FF" fill-opacity="0.4"/>
```

Comment:
same here

Reply:
@hanlinc27 Initially I had tried that, but for whatever reason the text would not appear in the blob SVG (assuming you mean doing it the same way I rendered the other 2 SVGs in the file). If it's more of an "unorganized" issue, I can create another js file strictly for the 'blob' and then just import it in the ThankYouSection.js. I believe that should not give me any issues. What are your thoughts?

Reply:
Ok that makes sense to me. I just find the readability a bit hard with the huge chunk of pathing but it should be fine.